configs/components/pxp-agent.rb

```diff
@@ -103,7 +103,9 @@
   when "aix"
     pkg.install_service "resources/aix/pxp-agent.service", nil, "pxp-agent"
   when "windows"
-    puts "Service files not enabled on windows"
+    # Note - this definition indicates that the file should be filtered out from the Wix
+    # harvest. A corresponding service definition file is also required in resources/windows/wix
+    pkg.install_service "SourceDir\\ProgramFiles64Folder\\PuppetLabs\\Puppet\\bin\\nssm.exe"
```

Comment:
@melissa are you talking about the directory structure above the .exe s? The precedent set by the other platforms is to leave the entire path in. unless we wanted to change all I would say no variable